myhpi/core/templates/core/information_page.html

```diff
@@ -17,9 +18,22 @@ <h1 class="page-title">
         <div class="side-panel-container">
             <aside class="side-panel border-accent">
                 <h1 class="toc-title">{% translate "Last edited" %}</h1>
-                {{ page.last_published_at|date:"d.m.Y, H:i" }}
+                {% get_current_timezone as TIMEZONE %}
+                <time id="last-published" datetime="{{ page.last_published_at|date:"c" }}" title="{{ TIMEZONE }}">
+                    <span id="last-published-date">
+                        {{ page.last_published_at|localtime|date:"SHORT_DATE_FORMAT" }}
+                    </span>
+                    {% translate "at" %}
+                    <span id="last-published-time">
+                        {{ page.last_published_at|localtime|time:"TIME_FORMAT" }}
+                    </span>
+                {% get_current_language as LANGUAGE_CODE %}
+                {% if LANGUAGE_CODE != "en" %}
```

Comment:
You don't need to think about the current_language in templates. Just use the `translate` template tag with the english value and django will take care of displaying the correctly translated output

Reply:
You could use a blocktranslate tag and then use the value inside of that. In the german version you could then just add the "Uhr" behind the value